clients/drcachesim/analyzer_multi.cpp

```diff
@@ -450,7 +450,8 @@ analyzer_multi_tmpl_t<RecordType, ReaderType>::analyzer_multi_tmpl_t()
                 nullptr, op_verbose.get_value(), op_jobs.get_value(),
```

Comment:
> Overall, for a large app, we were still able to get >98% of the kernel traces converted successfully. 

Seems worth mentioning that these imperfect traces will result in invariant checker errors about both A) PC discontinuities in syscall traces and B) missing syscall traces.

Reply:
In the PR description? It does say that; in the paragraphs above that sentence.